.gitignore

```diff
@@ -22,3 +22,5 @@ bin/tmp
 vendor
 .tarball
 go-ipfs-source.tar.gz
+
+.idea
```

Comment:
I'd recommend putting this in `$HOME/.gitignore`, project specific gitignores shouldn't really have to care about all IDEs in existence.

Reply:
Will do!